pysmt/smtlib/parser/parser.py

```diff
@@ -1243,8 +1242,15 @@ def _cmd_get_objectives(self, current, tokens):
         self.parse_atoms(tokens, current, 0)
         return SmtLibCommand(current, [])
 
+    def _cmd_minmax_maxmin_obj(self, current, tokens):
+        """(minmax | maxmin (<term>+) [:id <string>] [:signed])"""
+        params = self.parse_expr_list(tokens, current)
+        self.consume_closing(tokens, current)
```

Comment:
Add a todo or something because here you are not supporting the `:signed` nor the `:id` options...